src/vs/workbench/contrib/chat/browser/actions/chatAccessibilityHelp.ts

```diff
@@ -27,23 +24,16 @@ export function getAccessibilityHelpText(accessor: ServicesAccessor, type: 'chat
-				content.push(keybinding ? localize('inlineChat.diff', "Tab again to enter the Diff editor with the changes and enter review mode with ({0}). Use Up/DownArrow to navigate lines with the proposed changes.", keybinding) : localize('inlineChat.diffNoKb', "Tab again to enter the Diff editor with the changes and enter review mode with the Go to Next Difference Command. Use Up/DownArrow to navigate lines with the proposed changes."));
-				content.push(localize('inlineChat.acceptReject', "Tab again to reach the action bar, which can be navigated with Left/RightArrow."));
-			}
-		} else if (command === 'explain') {
-			content.push(localize('inlineChat.explain', "/explain commands will be run in the chat view."));
-			content.push(localize('inlineChat.chatViewFocus', "To focus the chat view, run the GitHub Copilot: Focus on GitHub Copilot View command, which will focus the input box."));
-		} else {
-			content.push(localize('inlineChat.toolbar', "Use tab to reach conditional parts like commands, status message, message responses and more."));
-		}
+		const startChatKeybininding = keybindingService.lookupKeybinding('inlineChat.start')?.getAriaLabel();
+		content.push(localize('inlineChat.overview', "Inline chat occurs within a code editor and takes into account current selection. It is useful for refactoring, fixing, and more. Keep in mind that Copilot generated code may be incorrect."));
+		content.push(localize('inlineChat.access', "It can be activated via the Fix and Explain with Copilot context menu actions or directly using the command: Inline Chat: Start Code Chat {0}.", startChatKeybininding));
+		content.push(localize('chat.requestHistoryInline', 'In the input box, use UpArrow/DownArrow to navigate your request history. Edit input and use enter or the make request button to run a new request.'));
+		content.push(localize('inlineChat.contextActions', "Explain and Fix with Copilot actions run a request prefixed with /fix or /explain. These prefixes can be used directly in the input box to apply those specific actions."));
+		content.push(localize('inlineChat.fix', "When a request is prefixed with /fix, a response will indicate the problem with the current code. A diff editor will be rendered and can be reached by tabbing."));
+		const diffReviewKeybinding = keybindingService.lookupKeybinding('editor.action.diffReview.next')?.getAriaLabel();
+		content.push(diffReviewKeybinding ? localize('inlineChat.diff', "Once in the diff editor, enter review mode with ({0}). Use Up/DownArrow to navigate lines with the proposed changes.", diffReviewKeybinding) : localize('inlineChat.diffNoKb', "Tab again to enter the Diff editor with the changes and enter review mode with the Go to Next Difference Command. Use Up/DownArrow to navigate lines with the proposed changes."));
+		content.push(localize('inlineChat.explain', "When a request is prefixed with /explain, a response will explain the code in the current selection and the chat view will be focused."));
```

Comment:
> and the chat view will be focused

Is this needed? This is talking about /explain in the input box (where it will be focused), not the Explain action specifically.

Reply:
This is what I'm describing 
![Screenshot 2023-06-12 at 4 08 05 PM](https://github.com/microsoft/vscode/assets/29464607/883f300c-9919-46fe-8b47-f9d81ac1af41)

Reply:
it's needed because they need to understand that running that will move their focus away from the editor